lexicons/tools/ozone/set/defs.json

```diff
@@ -0,0 +1,49 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.defs",
+  "defs": {
+    "set": {
+      "type": "object",
+      "required": ["name"],
+      "properties": {
+        "name": {
+          "type": "string",
+          "minLength": 3,
+          "maxLength": 128
+        },
+        "description": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 1024
+        }
```

Comment:
It's a nit, but for internationalization we are often doing "descriptions" as a grapheme limit, then 10x that as maxLength. so:

```
maxGraphemes: 1024
maxLength: 10240
```

and then update in the views/references to this field